src/pages/Cores/Home/index.tsx

```diff
@@ -0,0 +1,42 @@
+import {getCores} from 'selectors/state';
+import {SFC} from 'types';
+import {useMemo} from 'react';
+import {useSelector} from 'react-redux';
+import {useToggle} from 'hooks';
+import * as S from './Styles';
+import CoreCard from '../CoreCard';
+import CoreModal from 'modals/CoreModal';
+import EmptyText from 'components/EmptyText';
+import orderBy from 'lodash/orderBy';
+import SectionHeading from 'components/SectionHeading';
```

Comment:
As stated in the previous PR reviews, can we please keep up the imports with the convention?

What I have understood is we have the following imports ordering:

1. React, Redux imports
2. External/third party libraries imports
3. Internal imports
4. Style import

So keeping this order in mind, we will have something like this:
```
import {useMemo} from 'react';
import {useSelector} from 'react-redux';
import orderBy from 'lodash/orderBy';

import CoreCard from '../CoreCard';
import CoreModal from 'modals/CoreModal';
import EmptyText from 'components/EmptyText';
import SectionHeading from 'components/SectionHeading';
import {SFC} from 'types';
import {getCores} from 'selectors/state';
import {useToggle} from 'hooks';

import * as S from './Styles';
```